frontend/src/components/layout/Header.js

```diff
@@ -388,7 +391,13 @@ function OEHeader(props) {
                           {userSessionDetails.firstName}{" "}
                           {userSessionDetails.lastName}
                         </li>
-                        <li
+                         <li
+                         onClick={()=>redirectToApplicationProperties()}
+                          className="userDetails clickableUserDetails"
+                        > 
+                          Application Properties
```

Comment:
Text should use the internationalization functionality available by using intl.formatMessage({}), providing a message key, and placing the english value in en.json. We also support French (fr.json), but that can be added independently of this commit.

Also move this to the admin Menu ie [here](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/frontend/src/components/admin/Admin.js#L75-L77) and add a Route there ie [this](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/frontend/src/components/admin/Admin.js#L113-L115)